manifests/network/openshift-sdn/001-rbac.yaml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sdn
```

Comment:
since this is non-namespaced should it have a more distinctive name?

Reply:
definitely, good point!